test/helpers/statsd_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module StatsDHelper
+  extend self
```

Comment:
Why make this a class method? I think our other helpers use the model of making the test classes that need them `include` the helper's class.